src/main/java/com/oramon/kata/GameOfLife.java

```diff
@@ -0,0 +1,29 @@
+package com.oramon.kata;
+
+import com.oramon.kata.draw.DrawStrategy;
+import com.oramon.kata.draw.impl.DrawStrategyImpl;
+
+/**
+ * ClassName
+ * <p>
+ * Created by Alfredo Gil Rubio
+ * <p>
+ * Date: 26/05/17 11:57
+ */
+public class GameOfLife {
+    private int[][] world;
```

Comment:
At this point I think is ok, when I decide to introduce a new world I will do this decision.